js/main.js

```diff
@@ -54,7 +54,7 @@
             }
         });
 
-        $('.is-sortable').sortable({
+        $('div.is-sortable').sortable({
```

Comment:
Guessing not based on the CSS further down, so maybe remove the `form__group` class from the placeholder?